cluster/common.sh

```diff
@@ -20,12 +20,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-KUBE_ROOT=$(cd $(dirname "${BASH_SOURCE}")/.. && pwd)
```

Comment:
KUBE_ROOT is getting set to wrong path, and same path is used for both Master and Minion node.